extensions/default/src/commandsModule.js

```diff
@@ -1,10 +1,21 @@
-const commandsModule = ({ servicesManager }) => {
-  const { MeasurementService } = servicesManager.services;
+const commandsModule = ({ servicesManager, commandsManager }) => {
+  const { MeasurementService, ViewportGridService, ToolBarService } = servicesManager.services;
 
   const actions = {
     clearMeasurements: () => {
       MeasurementService.clear();
     },
+    toggleCine: () => {
+      const { viewports, isCineEnabled } = ViewportGridService.getState();
+      ViewportGridService.setIsCineEnabled(!isCineEnabled);
+      ToolBarService.setButton('Cine', { props: { isActive: !isCineEnabled } });
+      viewports.forEach((vp, index) => {
+        ViewportGridService.setCineForViewport({
```

Comment:
We need to not polute the viewport grid service with methods specific to a viewport type.
The viewport grid service should control nothing but the viewport grid layout.

For this we should use a specific service or context (see the tracking context as an example), inside the measurement-tracking extension, that the viewport can listen for to set its UI.

cc @dannyrb @swederik you agree?

Reply:
Thinking about this, we want the CINE to be reusable, so we should do this inside the cornerstone extension, and the  measurement tracking viewport should just subscribe to that service (so that the cornerstone extension's viewport could just do the same).

@dannyrb I guess we need to listen to services/context cross extension, we need to think about dependency management more concretely at some point? 🤔

Reply:
@JamesAPetts https://precisionmetrics.atlassian.net/browse/OHIF-323 I decided to go with the approach in the card description. @dannyrb @JamesAPetts can we document the viewport grid service/all services and its final purpose? sometimes I found it hard to keep up with all design decisions made previously in OHIF and the atomicity of each service.

Reply:
I know you guys have been thinking a lot about these services together, maybe we can set up a call to get me up to speed with design decisions?